src/__tests__/resources/assets.unit.spec.ts

```diff
@@ -198,4 +199,49 @@ describe('Asset unit test', () => {
       });
     });
   });
+
+  describe('shallow copy + JSON.stringify()', () => {
+    const items = [
+      {
+        id: 1,
+      },
+      {
+        id: 2,
+      },
+    ];
+
+    test('assets.list().autoPagingToArray()', async () => {
+      nock(mockBaseUrl)
+        .post(new RegExp('/assets/list'), {})
+        .once()
+        .reply(200, { items });
+
+      const assets = await client.assets.list().autoPagingToArray({ limit: 2 });
+      expect(JSON.stringify(assets)).toBeTruthy();
```

Comment:
`toBeTruthy` will I consider to be a weak test here. What do you test with that?